samples/bootloader/src/main.c

```diff
@@ -189,6 +221,21 @@ void main(void)
 	}
 #endif /* CONFIG_SOC_NRF9160 */
 
-	boot_from((u32_t *)s0_address_read());
+	const struct image_version *s0_ver = mcuboot_get_version(PM_S0_PAD_ADDRESS);
+	const struct image_version *s1_ver = mcuboot_get_version(PM_S1_PAD_ADDRESS);
+
+	if (s0_ver == NULL && s1_ver == NULL) {
+		printk("No bootable image found aborting...\n\r");
+		return;
+	}
+
+	if (mcuboot_is_version_sufficient(s1_ver, s0_ver) == 0
```

Comment:
One of the versions could still be NULL during these comparisons.

Reply:
Should be fixed this comment is outdated.